Packs/Cybersixgill-ActionableAlerts/IncidentFields/Cybersixgill_Suspicious_Domain.json

```diff
@@ -0,0 +1,31 @@
+{
+    "id": "incident_suspiciousdomain",
```

Comment:
Please add a prefix to the file name incidenttype-Cybersixgill_Suspicious_Domain.json

Reply:
Done